.github/workflows/import-data-from-ga-news.yml

```diff
@@ -1,19 +1,19 @@
-name: GA Data Importer cj-test-17
+name: GA Data Importer news
```

Comment:
Can we add some more detail to this name like "marketing-site-news" or something similar? That way someone may have more context in case of troubleshooting.